src/components/QuranReader/ReadingPreferenceSwitcher/ReadingPreferenceSwitcher.module.scss

```diff
@@ -11,6 +12,7 @@ $contextMenuReadingPreferenceMaxWidth: calc(10 * var(--spacing-large));
 }
 
 .surahHeaderContainer {
+  padding-top: $readingPreferenceSwitcherContainerTopPadding;
```

Comment:
This will apply to all Musahfs and not just Tajweed.

| Before    | After |
| -------- | ------- |
| <img width="1728" alt="Screenshot 2024-06-21 at 10 31 42 AM" src="https://github.com/quran/quran.com-frontend-next/assets/15169499/4850d7c4-c58d-4b8e-9e95-cfd8d688fa3e">  | <img width="1728" alt="Screenshot 2024-06-21 at 10 31 58 AM" src="https://github.com/quran/quran.com-frontend-next/assets/15169499/7b6f99ee-c18d-418c-9b8d-39984d585ead">   |